src/components/LayoutColumns/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import {map} from 'lodash'
+
+export default ({items}) => (
+  <div className='flex-ns'>
+    {map(items, (item, index) => (
+      <div
+        key={index}
+        className={`w-100-ns ${index < items.length - 1 ? 'mr4-ns' : ''}`}
+      >
+        {item}
+      </div>
+    ))}
```

Comment:
This component lets you pass a list of `items` and it makes them into a horizontal grid on larger screen sizes with margin "gutter" between, except on the last item.